modules/ROOT/pages/branded-emails.adoc

```diff
@@ -0,0 +1,66 @@
+= Optionally Configure Branded Emails
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager sends notification emails when an API contract
+is approved or revoked. By default these emails use Anypoint addresses and
```

Comment:
```suggestion
is approved or revoked. By default these emails use Anypoint Platform addresses and
```

Reply:
Global: I think we have to use the full product name.

Reply:
You're exactly right-- just checked the style guide and it says "Anypoint Platform", not "the Anypoint Platform" and not just "Anypoint". I probably need to clean up a lot of those two wrong ones in the ACM and Exchange docs. Thank you for the reminder.